### ISSUE TITLE
tweedledum+dee's small button height makes the UI look bad

### ISSUE THREAD

**Issue by cgolubi1** [buttonmen-dev/buttonmen#1950]
What it says on the tin:

![tweedledum_dee](https://cloud.githubusercontent.com/assets/2898207/13028864/92effd36-d249-11e5-8a2c-3457d1eda5a7.png)

**Comment by cgolubi1** [comment]
We could have the UI code put every image in a box of the right size somehow, or we could have the art redone so that it is itself in a transparent box of the right size.  IMO, the second one of those sounds easier from a UI maintenance point of view.  It doesn't seem unreasonable to expect that all button art files should be square.

**Comment by AdmiralJota** [comment]
I think we have two options for the art rule: expect that all buttons are square *or* expect that all buttons are a standard height. Because making this button image square wouldn't help us any unless we *also* made that square taller than the image is now.

**Comment by cgolubi1** [comment]
Sure, i was assuming a square of the current width of the image (which is similar to other art) and the same height as its width.  That is, i was assuming we'd add clear background to the top and bottom, rather than make the whole image smaller.

**Comment by AdmiralJota** [comment]
Right, I understand that was the intention for this image. I was just addressing the "It doesn't seem unreasonable to expect that all button art files should be square." part. I think that if we're going to set an expectation going forward, it's more important to set that expectation on the height of the images than the aspect ratio.

**Comment by irilyth** [comment]
I feel like "button images should be 150x150" would be a good expectation to set, especially since it's currently true for every button image other than _pepper, _zal, and tweedledumdee.

**Comment by blackshadowshade** [comment]
So, this issue has moved on a little bit, and we now have the potential of rectangular images, representing button character cards.

The fix mentioned in #2187 of changing Game.js, line 2818 (in Game.buttonImageDisplay) from width: 150 px to height: 150 px doesn't address the appearance of the Tweedles' image, since this is still constrained by the containing box.

**Comment by irilyth** [comment]
> I feel like "button images should be 150x150" would be a good expectation to set, especially since it's currently true for every button image other than _pepper, _zal, and tweedledumdee.

I still like this idea. :^ ) It wouldn't be hard to just resize the images in the repo; any reason not to just do that?

**Comment by blackshadowshade** [comment]
For cards, I have no problems with resizing them to 150 x 150 px, adding transparent pixels on both sides. However, for the Tweedles, I think that its current resolution of 150 x 230 px is already pretty poor, and making it worse is undesirable.

There are three places on the site where the size of the Tweedles could potentially be an issue:
- the play mat
- the button set page
- the button page

I think that changing the constraint on image size from width-restricted to height-restricted should fix both the button set page and the button page. The play mat appears to be a little more complex, but it should also be possible to tweak it to expand horizontally for the Tweedles.

**Comment by blackshadowshade** [comment]
I note that there is an alternative image of the Tweedle's that is actually round, in case anyone wants to change the image:

![](https://vignette.wikia.nocookie.net/buttonmen/images/0/08/TweedleDumTweedleDee.png/revision/latest?cb=20190925084753 "")
